PROPOSAL-DRAFT-V1.0.md

```diff
@@ -0,0 +1,279 @@
+# TSSG
+*Current Working Draft*
+
+## Introduction
+This is the specification for TSSG(The Swagger Schema Generator), which is a new and easy to understand Syntax/Grammar to write OpenAPI Schema in an easy, clean and concise way.
```

Comment:
```suggestion
This is the specification for TSSG(The Swagger Schema Generator), which is a new and easy to understand Syntax/Grammar to write OpenAPI Schema in an easy, clean and concise way.
```

This is the specification for TSSG(The Swagger Schema Generator), which is a new and easy to understand Syntax/Grammar that helps writing OpenAPI Schema in short, clean and concise way.

Reply:
This is the specification for TSSG(The Swagger Schema Generator), which is a new and easy **way** to understand Syntax/Grammar to write OpenAPI Schema in an easy, clean and concise way.

Reply:
Thank You @mingwho .  But I think TSSG is not a way to understand syntax/grammar to write OpenApi Schema. it has its own syntax/grammar. so IMO adding **way** will make it more of a tool to understand grammar. @tauqeernasir @heypran  what do you guys say?

Reply:
Thanks for the comment. I guess we can write it like this. What do you guys think?
```This is the specification for TSSG(The Swagger Schema Generator), which enables developers to generate OpenAPI Schema with an easy and concise syntax/grammar.```

Reply:
LGTM